modules/bonding-multiple-sriov-network-interfaces-to-dual-port.adoc

```diff
@@ -0,0 +1,135 @@
+endif::ibm-z,ibm-z-kvm[]
+ifdef::ibm-z,ibm-z-kvm[]
+You can use a network teaming as an alternative to bonding by using the `team=` parameter.
+endif::ibm-z,ibm-z-kvm[]
+
+.Procedure
+
+ifndef::ibm-z,ibm-z-kvm[]
+ifndef::installing-ibm-power[]
+. Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in the "Attaching SR-IOV networking devices to virtual machines" section.
+endif::installing-ibm-power[]
+ifdef::installing-ibm-power[]
+. Create the SR-IOV virtual functions (VFs).
+endif::installing-ibm-power[]
+
+. Create the bond, attach the desired VFs to the bond and set the bond link state up following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_networking/configuring-network-bonding_configuring-and-managing-networking[Configuring network bonding]. Follow any of the described procedures to create the bond.
++
+The following examples illustrate the syntax you must use:
++
+* The syntax for configuring a bonded interface is `bond=<name>[:<network_interfaces>][:options]`.
++
+`<name>` is the bonding device name (`bond0`), `<network_interfaces>` represents the virtual functions (VFs) by their known name in the kernel and shown in the output of the `ip link` command(`eno1f0`, `eno2f0`), and _options_ is a comma-separated list of bonding options. Enter `modinfo bonding` to see available options.
++
+* When you create a bonded interface using `bond=`, you must specify how the IP address is assigned and other information for the bonded interface.
++
+** To configure the bonded interface to use DHCP, set the bond's IP address to `dhcp`. For example:
++
+[source,terminal]
+----
+bond=bond0:eno1f0,eno2f0:mode=active-backup
+ip=bond0:dhcp
+----
++
+** To configure the bonded interface to use a static IP address, enter the specific IP address you want and related information. For example:
++
+[source,terminal]
+----
+bond=bond0:eno1f0,eno2f0:mode=active-backup
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
+----
+endif::ibm-z,ibm-z-kvm[]
+ifndef::ibm-power[]
+
+. As an optional task, you can use network teaming as an alternative to bonding by using the `team=` parameter.
++
+* The syntax for configuring a team interface is: `team=name[:network_interfaces]`
++
+_name_ is the team device name (`team0`) and _network_interfaces_ represents a comma-separated list of physical (ethernet) interfaces (`em1, em2`).
++
+[NOTE]
+====
+Teaming is planned to be deprecated when {op-system} switches to an upcoming version of {op-system-base}. For more information, see this https://access.redhat.com/solutions/6509691[Red Hat Knowledgebase Article].
+====
```

Comment:
If we have KCS articles that describe how teaming works, and it is planned to be deprecated, this seems like a strong case to not have it in the docs at all. 

This is only my opinion, but you might want to consider if it is worth continuing to carry this tech debt over.